src/parsers.js

```diff
@@ -1,6 +1,5 @@
 const { RequiredArg, Command } = require("./commands.js")
 const fs = require('fs')
-const { Console } = require("console")
```

Comment:
what is this line for?

Reply:
idk it was unused so i deleted it

Reply:
oh lol didn't even notice it was removed and not added, oops